tests/dataset-hash-collisions/test.yaml

```diff
@@ -0,0 +1,12 @@
+pcap: ../flowbit-oring/input.pcap
+
+requires:
+  features:
+    - HAVE_LIBJANSSON
+  files:
+    - src/datasets.c
```

Comment:
nit: these requires are not needed anymore

+ files requires are bad in case we move the datasets.c to rust

Reply:
Interesting insight. I had not thought about this. Thanks a lot! 🙇🏽‍♀️ 
Copy pasted this all from another test. Will change in the next rev